.github/workflows/ci.yaml

```diff
@@ -128,4 +128,8 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: nixbuild/nix-quick-install-action@v26
+      - uses: cachix/cachix-action@v12
```

Comment:
I think this should probably happen in the release workflow, so that it only caches new releases

Reply:
so that nix users are only getting stable releases, not a canary/nightly build, and I think this would also cache binaries for pull requests? I guess i'm not entirely sure how the cache works

Reply:
Cachix offers a decent amount of storage space for free (10 GiB at present), but if you want that level of control, it may be a good idea to use [magit-nix-cache-action](https://github.com/DeterminateSystems/magic-nix-cache-action/) instead for CI.

Reply:
Can you elaborate on how this saves us CI time? we are using setup-beam for the elixir/erlang installation and already cache mix dependencies

Reply:
It would save time on nix-build CI job I added in the previous PR. It's unnecessary if you don't intend to provide Nix packages from the flake. You are not using Nix to build the release assets, so the action would never accelerate your main workflow. Fro Nix, the action caches the Nix store, which saves time spent on rebuilding rebar3 with the latest Erlang every time.